### PR TITLE
[version] 8u121 b13

### PR DESCRIPTION
**Pull request by albundy83**
Hello,
just a demo on how I have fixed the new url link of oracle.
Tell me what you think.

### CODE REVIEW

File: generate_dockerfiles.sh [hunk 1]
```diff
@@ -53,6 +54,13 @@ for version in ${JAVA_VERSIONS[@]}; do
   JVM_MAJOR=$(echo $version | cut -d- -f1)
   JVM_MINOR=$(echo $version | cut -d- -f2)
   JVM_BUILD=$(echo $version | cut -d- -f3)
+  JVM_TEMP=$(echo $version | cut -d- -f4)
+
+  if [ "${JVM_MINOR}" -ge 121 ]; then
```
Comment (anapsix) [inline generate_dockerfiles.sh:59]: I see what you are doing.. cause they need to be difficult now..

**Review by anapsix — CHANGES_REQUESTED** [review]
that should work, however please update README.md to reflect latest version of Java and DCEVM

File: generate_dockerfiles.sh (hunk 1 above)
Comment (albundy83) [reply generate_dockerfiles.sh:59]: Yes not sure if it's very clean but it's not too ugly.

**Comment by anapsix** [comment]
thanks @albundy83!

**Pull request merged**